etc/profile-m-z/mutt.profile

```diff
@@ -34,15 +38,82 @@ noblacklist ${HOME}/sent
 blacklist /tmp/.X11-unix
 blacklist ${RUNUSER}/wayland-*
 
+include allow-perl.inc
+include allow-python2.inc
+include allow-python3.inc
```

Comment:
Are they needed? mutt.profile has `disable-interpreters.inc` since 3 years.

Reply:
These are some optional functionality for Oauth etc. I will comment them.